apps/site/assets/ts/hooks/useStop.ts

```diff
@@ -1,9 +1,11 @@
 import useSWR from "swr";
 import { fetchJsonOrThrow } from "../helpers/fetch-json";
-import { Stop } from "../__v3api";
+import { Facility, Stop } from "../__v3api";
 import { FetchState, FetchStatus } from "../helpers/use-fetch";
 
 const fetchData = async (url: string): Promise<Stop> => fetchJsonOrThrow(url);
+const fetchFacilityData = async (url: string): Promise<Facility[]> =>
+  fetchJsonOrThrow(url);
```

Comment:
nit-pick: Since these functions do the same thing we could try generics to combine them into one:
`const fetchData = async <T>(url: string): Promise<T> => fetchJsonOrThrow(url);`